stable/nginx/nginx/templates/NetworkPolicy.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: networking.k8s.io/v1
```

{{ if .Values.NetworkPolicy.Enabled }}
```suggestion
apiVersion: networking.k8s.io/v1
```